private/buf/bufcli/errors_tls_go119.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !go1.20
+
+package bufcli
+
+import (
+	"errors"
+	"strings"
+)
+
+// wrappedTLSError returns an unwrapped TLS error or nil if the error is another type of error.
```

Comment:
I'd like to see a bit more documentation here as to the motivation for this, so that in the future we understand why this is here in case we need to debug

Reply:
Ok - this is just a temporary thing until Go 1.21 is released and we can remove and just use errors.As which will be a lot more readable. I'll update to add more details though.